tests/onnx/quantization/test_detection_models_graph.py

```diff
@@ -17,37 +17,79 @@
 
 import onnx
 
+from nncf.experimental.onnx.model_normalizer import ONNXModelNormalizer
+from nncf.experimental.onnx.algorithms.quantization.utils import find_ignored_scopes
+
 from tests.common.helpers import TEST_ROOT
 
 from tests.onnx.quantization.common import min_max_quantize_model
 from tests.onnx.quantization.common import compare_nncf_graph
 from tests.onnx.quantization.common import infer_model
 
 MODELS_NAME = [
+    'ssd-12',
     'yolov2-coco-9',
     'tiny-yolov2',
+    'MaskRCNN-12',
+    'retinanet-9',
+    'fcn-resnet50-12'
 ]
 
 PATH_REF_GRAPHS = [
+    'ssd-12.dot',
     'yolov2-coco-9.dot',
     'tiny-yolov2.dot',
+    'MaskRCNN-12.dot',
+    'retinanet-9.dot',
+    'fcn-resnet50-12.dot'
 ]
 
 INPUT_SHAPES = [
+    [1, 3, 1200, 1200],
     [1, 3, 416, 416],
     [1, 3, 416, 416],
+    [3, 30, 30],
+    [1, 3, 480, 640],
+    [1, 3, 480, 640],
+]
+
+DISSALOWED_OP_TYPES = [
```

Comment:
This model contains many layers that should be placed in ignored_scopes. To do that this is necessary to manually analyze the model graph, which is quite big. Because of that reason the ignored_types were added. 
It leads to not fully correct quantizers placements, but we could quantize the main computational part of the model - Convolutions.
I suggest to add this model in test with this option for now and mark it as not fully quantized and then when we have time to work on it.